scripts/newinstall.sh

```diff
@@ -33,8 +33,10 @@ EUPS_TARURL=${EUPS_TARURL:-"https://github.com/RobertLuptonTheGood/eups/archive/
 
 EUPS_PKGROOT=${EUPS_PKGROOT:-"http://sw.lsstcorp.org/eupspkg"}
 
-MINICONDA2_VERSION=${MINICONDA2_VERSION:-4.2.12.lsst2}
-MINICONDA3_VERSION=${MINICONDA3_VERSION:-4.2.12.lsst2}
+MINICONDA_VERSION=${MINICONDA_VERSION:-4.2.12}
+LSSTSW_REF=${LSSTSW_REF:-7c8e67}
```

Comment:
Can there be a comment here explaining that this ref is used to define the pinned package versions for conda? The main problem we have is that v13.0 of the stack wants different pinned versions to v12.0 and once you've installed a v13.0 stack your chances of building a v12.0 in the same tree are slim to none.